docs/source/integrating/asset-management.md

```diff
@@ -18,11 +18,12 @@ By default, Talk will use "Lazy Asset Creation" to dynamically generate Assets
 in Talk in order to make it easier for lighter installations. In order to have
 more strict control over this flow, we will create a plugin that will:
 
-1. Disable "Lazy Asset Creation" by [Overriding a Resolver](#overriding-a-resolver).
-2. Create Assets from our CMS by [Creating a New Asset Route](#creating-a-new-asset-route).
-3. Facilitate updates from our CMS to keep Talk in sync by [Creating an Asset Update Route](#creating-an-asset-update-route).
 
-We will then modify our embed so that we can [Target the Asset](#target-the-asset).
+1. Disable "Lazy Asset Creation" by [Overriding a Resolver](#Overriding%20a%20Resolver).
```

Comment:
Looking at the generated docs, it looks like the ID is actually: `#overriding-a-resolver`, same applies for the rest of these links.

Reply:
Links have been fixed, and confirmed on netlify preview